CONTRIBUTING.md

```diff
@@ -8,4 +8,30 @@ and watch out for more specific details in this file.
 
 Instead of directly editing `CHANGELOG.md`, please add a file `123.md`
 in `changelog-entries`, where `123` your pull request number. This helps reduce
-merge conflicts and we will merge these files at the time we release a new version.
\ No newline at end of file
+merge conflicts and we will merge these files at the time we release a new version.
+
+## Checking the Markdown files
+
+Every contribution is automatically checked for issues in the `.md` files.
+You can run these tests locally by installing [markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli)
+from npm:
+
+```bash
+npm install -g markdownlint-cli
+```
+
+If you get any permission errors when trying to install from npm, [set the npm prefix to a local directory](https://stackoverflow.com/a/54170648/2254346).
+
+After installing, you can check all files in the current directory with:
+
+```bash
+markdownlint .
+```
+
+There are also extensions for many editors that automatically check Markdown files with the same system.
```

Comment:
Not sure if this is the right way to go. I'm locally using `act` to execute github actions. This makes sure that I actually run locally what the github action also does. I would also recommend this approach to the user, because then we don't have to keep this documentation here synchronized with what is implemented in the github actions.

Note to avoid disappointment: For this specific `act` (of course) does not work. Trying to find out why.

Reply:
Ooops, I didn't know you intended to review. I did not know about [act](https://github.com/nektos/act), thanks!

In any case, the current way is very easy to run independently, and is works directly in the editor. But this is something we could look into and document further.

Reply:
> Ooops, I didn't know you intended to review. I did not know about [act](https://github.com/nektos/act), thanks!

No problem. Just a random remark from my side. `act` is not working with this action. Until this is fixed, it's not helpful and the documentation above is fine.

Reply:
I just found the problem. I removed the check of shell scripts in https://github.com/precice/fenics-adapter/commit/de476c36fa2fa0dff5947a25dbfc3c79eb2aba0c and now act works for me. I think I don't need this check and can therefore drop it. Alternatively: If you split the two checks, one for md and one for shell scripts, I assume the md part will work with act and you don't need to document it here.